### PR TITLE
Fixed a small typo ("om" to "from")

### PR DESCRIPTION
**Pull request by tupacko**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nunit/docs?pullRequest=521) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/nunit/docs?pullRequest=521) before we can accept your contribution.<br/><br/>:x: tupacko [sign now](https://cla.dotnetfoundation.org/nunit/docs?pullRequest=521)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/nunit/docs?pullRequest=521) it.</sub>

File: docs/articles/vs-test-adapter/Index.md [hunk 1]
```diff
@@ -4,16 +4,15 @@ The NUnit 3 Test Adapter allows you to run NUnit 3 tests inside Visual Studio. T
 
 It is not possible to run NUnit 2.x tests using this adapter. Use the original adapter for that purpose. If you need to work with projects using NUnit 2.x and other projects using NUnit 3, you may install both versions of the adapter.
 
-The current release is designed to work with Visual Studio 2012, 2013, 2015, 2017 and 2019. Some features are not available under VS2012 RTM.  It also works om command line using either vstest.console or dotnet test, and is also used by JetBrains ReSharper for Visual Studio and Rider IDE, and their corresponding command line tool.
+The current release is designed to work with Visual Studio 2012, 2013, 2015, 2017 and 2019. Some features are not available under VS2012 RTM. It also works from command line using either vstest.console or dotnet test, and is also used by JetBrains ReSharper for Visual Studio and Rider IDE, and their corresponding command line tool.
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Index.md:7]:
```suggestion
The current release is designed to work with Visual Studio 2012, 2013, 2015, 2017 and 2019. Some features are not available under VS2012 RTM. It also works from the command line using either vstest.console or dotnet test, and is also used by JetBrains ReSharper for Visual Studio and Rider IDE, and their corresponding command line tool.
```

**Review by SeanKilleen — CHANGES_REQUESTED** [review]
Nice catch! I suggested a minor change to make the phrasing more correct.

**Review by mikkelbu — APPROVED** [review]

**Review by tupacko — COMMENTED** [review]
I also included the suggested changes.

**Comment by SeanKilleen** [comment]
Very nice -- thank you @tupacko!

**Pull request merged**